config/master/manifests.jsonnet

```diff
@@ -172,7 +173,7 @@ local awsnode = {
               volumeMounts: [
                 {mountPath: "/host/opt/cni/bin", name: "cni-bin-dir"},
                 {mountPath: "/host/etc/cni/net.d", name: "cni-net-dir"},
-                {mountPath: "/host/var/log", name: "log-dir"},
+                {mountPath: "/var/log/aws-routed-eni", name: "log-dir"},
```

Comment:
Can we keep the `/host/` part here so that both log files end up in the same host folder? With this config, `ipamd.log` will be inside the container only, in `/host/var/log/aws-routed-eni/`, and the `plugin.log` file from the host is visible as `/var/log/aws-routed-eni/plugin.log`. Makes it more clear that it's actually a host mounted directory.

Reply:
Yep, reverted this part of the PR to continue using an explicit `/host` prefix.

Fwiw, I changed the `AWS_VPC_K8S_CNI_LOG_FILE` and `mountPath` in this PR to the same value so `ipamd.log` _was_ still writing to the same real host `/var/log/aws-routed-eni/`.  I was trying to _reduce_ confusion by using identical paths inside and outside the container, but obviously that didn't work :stuck_out_tongue:

Reply:
Ahh... Thanks. Was too confusing for me for sure 😆